client/keys/migrate.go

```diff
@@ -35,16 +37,18 @@ It is recommended to run in 'dry-run' mode first to verify all key migration mat
 		RunE: runMigrateCmd,
 	}
 
+	cmd.Flags().String(flagOldHome, "", "The root directory of the old keyring")
 	cmd.Flags().Bool(flags.FlagDryRun, false, "Run migration without actually persisting any changes to the new Keybase")
 	return cmd
 }
 
 func runMigrateCmd(cmd *cobra.Command, args []string) error {
+	oldRootDir, _ := cmd.Flags().GetString(flagOldHome)
```

Comment:
Good one @amaurym. I was wondering whether this should be an argument instead of an option?

Reply:
+1 for making it required actually. Right now the default value is `""`, which is incorrect if forgotten.

Reply:
https://github.com/cosmos/cosmos-sdk/pull/8639/commits/e10efd0e5721753342822b220a4434d99fe5d466